security/docker/Dockerfile.istio-ca-short

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:xenial
+
+ADD istio_ca /usr/local/bin/istio_ca
+
+ENTRYPOINT [ "/usr/local/bin/istio_ca", \
+"--self-signed-ca", \
+"--cert-ttl", "60s" ]
```

Comment:
Can you make this shorter, such as 20 sec?

Reply:
If cert-ttl is too short, then root certificate can be expired before the test ends.

Reply:
Actually I don't think you need to build a separate image just to change a value for an option. You can use the same image but adding/modifying/overwriting CLI options via the pod spec. See [here](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#define-a-command-and-arguments-when-you-create-a-pod).